config/config.go

```diff
@@ -64,6 +70,13 @@ func Get() *Config {
 	if v := os.Getenv("DOWNLOAD_SERVICE_URL"); len(v) > 0 {
 		cfg.DownloadServiceURL = v
 	}
+	if v := os.Getenv("ENABLE_DATASET_PREVIEW"); len(v) > 0 {
```

Comment:
While you are here is it worth migrating this app to use the Kelsey Hightower env config lib? It will make this less ugly.